extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,372 @@
+have any value it is RECOMMENDED that it be between 0 and 360 or +/- 180.
+
+|name|required|type|units|description|
+|----|--------|----|-----|-----------|
+|`azimuth`|false|float|degrees|Azimuth component of the direction in degrees increasing clockwise.|
+|`elevation`|false|float|degrees|Elevation component of the direction in degrees above horizon.|
+|`range`|false|float|meters|Line-of-sight slant range to emitter, if known, in meters.|
+|`az_error`|false|float|degrees|Error or uncertainty in the azimuth component.|
+|`el_error`|false|float|degrees|Error or uncertainty in the elevation component.|
+|`range_error`|false|float|meters|Error or uncertainty in the range component.|
+
+The `az_error`, `el_error`, and `range_error` field units are degrees, but the
+exact meaning of 'error' in this context is not explicitly defined. Applications
+SHOULD specify their specific meaning, and in general this should be interpreted
+as an uncertainty range. The error fields should be SHOULD NOT be used if the
+corresponding field estimates are not included.
+
+```json
+  "bearing": {
+    "azimuth": 211.2,
+    "elevation": 15.0,
+    "range": 30,
+    "az_error": 0.1,
+    "el_error": 0.1,
+    "range_error": 0.1
+  }
+```
+
+### 0.2 The `cartesian_point` Object
+
+The `cartesian_point` field is used to describe a single point in the Cartesian
+coordinate reference system. This object is necessary to define the phase center
+geometries for multidimensional arrays, and other cartesian locations.
+
+|name|required|type|units|description|
+|----|--------|----|-----|-----------|
+|`point`|true|float array|meters|Cartesian Point defined by a element array of [x,y,z] referenced to the `spatial` CRS.|
+
+
+## 1 Global
+
+The `spatial` extension adds the following fields to the `global` SigMF object:
+
+|name|required|type|units|description|
+|----|--------|----|-----|-----------|
+|`num_elements`|true|int|N/A|Defines the number of phase centers / channels collected in the Collection or multichannel Dataset.|
+|`channel_num`|true|int|N/A|The channel number, shares index with `element_geometry`.|
+
+The number of elements MUST be defined here and is constant for a given
+Collection. It may be tempting to use the `core:num_channels` field however
+that field specifies how many interleaved channels are present in a single
+Dataset whereas spatial Recordings may be spread over several individual
+Datasets in a SigMF Collection.
+
+In the case of a multichannel dataset, the `channel_num` specifies the first
+channel in the dataset. If all data is contained within that dataset then the
+`channel_num` field MUST be equal to zero.
+
+## 2 Captures
+
+The `spatial` extension adds the following fields to `captures` segment objects:
+
+|name|required|type|units|description|
+|----|--------|----|-----|-----------|
+|`element_geometry`|true|`cartesian_point` array|meters|Defines the relative arrangement of the array.|
```

Comment:
"`cartesian_point` array" should be redundant if "cartesian_point" is described as an array in its datatype description``